test/environment-test.js

```diff
@@ -699,19 +699,38 @@ COMMIT_MESSAGE:A shiny new feature`);
       });
     });
 
-    context('in Pull Request build', function() {
-      beforeEach(function() {
-        environment._env.SYSTEM_PULLREQUEST_PULLREQUESTNUMBER = '512';
-        environment._env.SYSTEM_PULLREQUEST_SOURCECOMMITID = 'azure-pr-commit-sha';
-        environment._env.SYSTEM_PULLREQUEST_SOURCEBRANCH = 'azure-pr-branch';
+    describe('Pull Request build', function() {
+      context('in build triggered by Git', function() {
+        beforeEach(function() {
+          environment._env.SYSTEM_PULLREQUEST_PULLREQUESTID = '512';
+          environment._env.SYSTEM_PULLREQUEST_SOURCECOMMITID = 'azure-pr-commit-sha';
+          environment._env.SYSTEM_PULLREQUEST_SOURCEBRANCH = 'azure-pr-branch';
+        });
+
+        it('has the correct properties', function() {
+          assert.strictEqual(environment.pullRequestNumber, '512');
+          assert.strictEqual(environment.branch, 'azure-pr-branch');
+          assert.strictEqual(environment.targetBranch, null);
+          assert.strictEqual(environment.commitSha, 'azure-pr-commit-sha');
+          assert.strictEqual(environment.targetCommitSha, null);
+        });
       });
 
-      it('has the correct properties', function() {
-        assert.strictEqual(environment.pullRequestNumber, '512');
-        assert.strictEqual(environment.branch, 'azure-pr-branch');
-        assert.strictEqual(environment.targetBranch, null);
-        assert.strictEqual(environment.commitSha, 'azure-pr-commit-sha');
-        assert.strictEqual(environment.targetCommitSha, null);
+      context('in build triggered by GitHub', function() {
+        beforeEach(function() {
+          environment._env.SYSTEM_PULLREQUEST_PULLREQUESTID = '512';
+          environment._env.SYSTEM_PULLREQUEST_PULLREQUESTNUMBER = '524';
```

Comment:
I'm unsure if we'll need to add another variable to the CLI for the case a GitHub PR triggers an Azure DevOps build. From my tests so far `env.SYSTEM_PULLREQUEST_PULLREQUESTNUMBER` has always been `null`
> This variable is populated for pull requests from GitHub which have a different pull request ID and pull request number.